contents/docs/integrate/_snippets/install-react-native.mdx

```diff
@@ -99,26 +89,37 @@ export function MyApp2() {
 You can further customize how PostHog works through its configuration on initialization.
 
 ```ts
-export const posthog = await PostHog.initAsync("<ph_project_api_key>", {
-    // PostHog API host, usually 'https://app.posthog.com' or 'https://eu.posthog.com'
-    host?: string = "<ph_instance_address>", // TIP: host is optional if you use https://app.posthog.com
```

Comment:
I'd keep the comment and `TIP`, this helped reduce questions from the community or customers.